accesscontroltool-bundle/src/main/java/biz/netcentric/cq/tools/actool/authorizableinstaller/impl/AuthorizableInstallerServiceImpl.java

```diff
@@ -70,14 +68,14 @@
     ExternalGroupInstallerServiceImpl externalGroupCreatorService;
 
     @Reference(cardinality = ReferenceCardinality.OPTIONAL, policy=ReferencePolicy.DYNAMIC, policyOption = ReferencePolicyOption.GREEDY)
-    volatile CryptoSupport cryptoSupport;
+    volatile AemCryptoSupport cryptoSupport;
```

Comment:
I tried without, it turns out this is needed because of the exception import `com.adobe.granite.crypto.CryptoException` (if if was the service `com.adobe.granite.crypto.CryptoSupport` alone, it would work without, but the exception class is required to e.g. load `AceBeanInstallerIncremental` if it is referenced there)

Reply:
It is only necessary if this checked exception is mentioned in a method signature. Please make sure to remove it there and instead rely on an unchecked exception or wrap in another checked exception.